integration/kubetoken/kubetoken_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build integration
+// +build integration
+
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
+
+import (
+	"encoding/json"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/okteto/okteto/integration"
+	"github.com/okteto/okteto/integration/commands"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+// TestKubetokenHasExpiration test that kubernetes returns a dynamic token with a set expiration
+func TestKubetokenHasExpiration(t *testing.T) {
+	t.Parallel()
+
+	oktetoPath, err := integration.GetOktetoPath()
+	require.NoError(t, err)
+
+	out, err := commands.RunOktetoKubetoken(oktetoPath, "")
```

Comment:
what if the testing server does not include this feature? :/ not sure if this is a possible scenario as this will be always available from 1.27... just wondering

Reply:
there was a similar discussion on: https://github.com/okteto/okteto/pull/3884#pullrequestreview-1562987362

my understanding is that the dynamic token is the default way of getting a token for the user so I don't see how this endpoint can not be there, especially because this has been part of the API for a few versions already.

Reply:
Could you review from which backend version we have that endpoint? For the tests It's fine to assume that the endpoint would be available. In normal flows of the CLI we should default to static token if the endpoint is not available. We cannot assume that customers have updated the backend to the latest version and some of them might be running an older version

Reply:
It seems it went out in our chart release `1.6.0`.